src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.searchbar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.searchbar-big::before {
+  content: "";
+  background-image: url(images/Search.svg);
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.searchbar-small::before {
+  content: "";
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
```

Comment:
there are some duplicated styles. could you please merge them by using the common class?